website/pages/en/cookbook/subgrounds-quickstart.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Quickstart to Subgrounds - Querying Subgraphs with Python
+---
+
+_TL;DR - [Subgrounds](https://github.com/0xPlaygrounds/subgrounds), built by [Playgrounds](https://playgrounds.network/), is an easy-to-use Python tool for interfacing with subgraphs._
+
+Countless teams and individuals building in web3 leverage fast, reliable, and unstoppable data pipelines that are powered by The Graph. Using a standard GraphQL API, anyone can query a subgraph to load blockchain data into their app’s frontend. This data supports a broad array of use cases such as DeFi, RWAs, DAOs, Gaming, NFTs, and more.
+
+[Playgrounds](https://playgrounds.network/) is unlocking new ways to leverage The Graph for another critical use case: analytics. Data analysts, who primarily work with structured data, can significantly benefit from the organized data served by Indexers on The Graph. However, while GraphQL as a query language is perfectly suited for web applications, like dapps, it’s less optimal for a variety of data analytics use cases.
+
+This is where Python comes in! Python has an incredible ecosystem of powerful analytics tools, making it the go-to language for anyone who is interested in exploring, interpreting, or visualizing data. Additionally, with the rise of Large Language Models (LLMs), integrating blockchain data into Python workflows has become even more crucial. Until recently, however, there was no streamlined way to connect subgraph data to a Python environment.
+
+---
+
+## Step 1: Install Subgrounds
+
+[Subgrounds](https://github.com/0xPlaygrounds/subgrounds) is an open source and intuitive Python library that is readily available to be used alongside powerful and well-known data analysis tools like [pandas](https://github.com/pandas-dev/pandas), [matplotlib](https://github.com/matplotlib/matplotlib), and [numpy](https://github.com/numpy/numpy).
+
+```bash
+pip install --upgrade subgrounds
+# or
+python -m pip install --upgrade subgrounds
+```
+
+Note that subgrounds requires Python 3.10 or higher and is available on [pypi](https://pypi.org/project/subgrounds/).
+
+## Step 2: Import Subgrounds and Set Your API Key
+
+Subgrounds allows interfacing with all subgraph endpoints, including both subgraphs in [Subgraph Studio](https://thegraph.com/studio/apikeys/) as well as subgraphs published to The Graph Network. To begin using subgrounds with The Graph Network, import the library and make sure you have a valid API key. You can generate an API key in Subgraph Studio to use it inside the subgraph’s query URL as shown [here](https://thegraph.com/docs/en/querying/querying-the-graph/#using-the-graph-explorer), or create a free Playgrounds API key. Currently Playgrounds offers 5,000 free queries per month.
```

Comment:
```suggestion
Subgrounds allows interfacing with all subgraph endpoints, including both subgraphs in [Subgraph Studio](https://thegraph.com/studio/apikeys/) as well as subgraphs published to The Graph Network. To begin using subgrounds with The Graph Network, import the library and make sure you have a valid API key. You can generate an API key in Subgraph Studio to use it inside the subgraph’s query URL as shown [here](/querying/querying-the-graph/#using-the-graph-explorer), or create a free Playgrounds API key. Currently Playgrounds offers 5,000 free queries per month.
```
Relative links ensure that users clicking on them in a translated version of the page don't land on an English page.

We should really have a lint rule for that. 😛 cc @dimaMachina @saihaj

Reply:
Though now that I think about it, the `#using-the-graph-explorer` hash wouldn't be valid in the translated page. Hmm... not sure how to handle that.